packit_service/service/api/usage.py

```diff
@@ -617,6 +621,92 @@ def get_result_dictionary(
     return {position_name: position, count_name: top_projects.get(project)}
 
 
+@usage_ns.route("/onboarded-projects")
+class Onboarded2024Q1(Resource):
+    @usage_ns.response(
+        HTTPStatus.OK,
+        "Onboarded projects for which exist a Bodhi update or a Koji build or a Packit merged PR.",
+    )
+    @classmethod
+    def calculate(cls):
+        known_onboarded_projects = GitProjectModel.get_known_onboarded_projects()
+
+        bodhi_updates = BodhiUpdateTargetModel.get_all_projects()
+        koji_builds = KojiBuildTargetModel.get_all_projects()
+        onboarded_projects = bodhi_updates.union(koji_builds).union(
+            known_onboarded_projects
+        )
+
+        sync_release_prs_urls = SyncReleaseTargetModel.get_all_downstream_pr_urls()
+        downstream_synced_projects_urls = {
+            pr[0 : pr.rfind("/pull-request")]  # noqa[203] prettier like it this way
+            for pr in sync_release_prs_urls
+        }
+        all_projects = GitProjectModel.get_all()
+        downstream_synced_projects = {
+            project
+            for project in all_projects
+            if project.project_url in downstream_synced_projects_urls
+        }
+        almost_onboarded_projects = downstream_synced_projects.difference(
```

Comment:
I was a little bit confused when I first read this, I am wondering what would be a better-expressing not that long name, could we have at least a comment explaining this term?